config/jobs/kubernetes/sig-cloud-provider/azure/OWNERS

```diff
@@ -2,19 +2,21 @@
 
 reviewers:
 - andyzhangx
-- brendandburns
 - CecileRobertMichon
 - feiskyer
+- jackfrancis
 - karataliu
```

Comment:
Does anyone know if @karataliu is still involved / should stay in the ONWERS file?